src/microbe_stage/BiomeConditions.cs

```diff
@@ -207,7 +207,14 @@ public void ApplyLongTermCompoundChanges(Biome biomeDetails, Dictionary<Compound
         foreach (var entry in changes)
         {
             var definition = simulationParameters.GetCompoundDefinition(entry.Key);
-            if (!definition.IsEnvironmental)
+            if (!definition.IsEnvironmental && !definition.IsGas)
```

Comment:
What's the purpose behind this change in this file?

Reply:
for temperature update, which can go under value of 0

Reply:
nevermind, I forgot that I started using a different mothed, I will revert that